cms/djangoapps/contentstore/views/item.py

```diff
@@ -266,6 +269,8 @@ def service(self, block, service_name):
                 return DjangoXBlockUserService(self._user)
             if service_name == "studio_user_permissions":
                 return StudioPermissionsService(self._user)
+            if service_name == "lti-configuration":
+                return ConfigurationService(LTIConsumerHideFieldsFlag)
```

Comment:
FYI - as an update on this thread:
@muzaffaryousaf reached out to me about implementing configuration settings for 2 xblocks (lti-consumer and video) and pointed out this thread.  I, independently, suggested that the xblocks use django directly to retrieve their own settings.  I would like for us to formalize this approach and now embrace allowing xblocks to use django directly.  (FYI @scottrish and @jbarciauskas)

The current implementation in this PR leads to long-term scalability and dependency issues:  with common CMS code needing to know about configuration implementation details/models of (a growing list of) xBlocks.  Rather, having each xBlock be self-contained and own its own models seems like the way to go.